upstream-community-operators/jaeger/jaeger.package.yaml

```diff
@@ -0,0 +1,4 @@
+packageName: jaeger
```

Comment:
Should we leave the packageName as is? If there is potential that the kubernetes operators and the community operators are added to the same cluster there would be naming collisions.

Reply:
Based on recent discussions I think the way we're going forward is that the version of the operator(openshift/k8s) that'll show up will depend on whether you're using okd/ocp/vanilla k8s.

> " Right now for lvl 1's, we have 3 community operators (camelk, jaeger, dynatrace) that should show up for OCP/OKD.  We have 1 (jaeger) that should show up on vanilla k8s".

If that's the approach we're going with I don't think it's necessary to modify the packageName.
Maybe @aravindhp and @tkashem could shed more light on this?

Reply:
This should be unchanged. Since k8s operators won't be available on the same cluster as the other operators there is no conflict.